backend/de.metas.business/src/main/java/de/metas/pricing/rules/campaign_price/callout/C_Campaign_Price.java

```diff
@@ -123,9 +123,17 @@ private static void updatePricingInfo(final I_C_Campaign_Price record)
 						.setCountryId(countryId)
 						.setPriceDate(TimeUtil.asLocalDate(record.getValidFrom(),timeZone))
 						.setFailIfNotCalculated();
-				final IPricingResult pricingResult = pricingBL.calculatePrice(pricingContext);
-				record.setC_TaxCategory_ID(pricingResult.getTaxCategoryId().getRepoId());
-				record.setInvoicableQtyBasedOn(pricingResult.getInvoicableQtyBasedOn().getCode());
+				final IPricingResult pricingResult;
+				try
+				{
+					pricingResult = pricingBL.calculatePrice(pricingContext);
+					record.setC_TaxCategory_ID(pricingResult.getTaxCategoryId().getRepoId());
+					record.setInvoicableQtyBasedOn(pricingResult.getInvoicableQtyBasedOn().getCode());
+				}
+				catch (final Exception e)
```

Comment:
I have mixed feelings about this approch, because
1. https://dzone.com/articles/exceptions-as-controlflow-in-java#:~:text=One%20of%20these%20common%20bad,makes%20your%20code%20less%20readable.
2. the exception is swallowed...

But before making my mind, I have to understand what exactly was behaving the system when this exception was thrown...
Is there a place where I can quickly reproduce it?